doc/rbd/iscsi-initiators.rst

```diff
@@ -12,9 +12,16 @@ Configuring the iSCSI Initiators
 
     .. warning::
 
-        Applications that use SCSI persistent group reservations (PGR) and
-        SCSI 2 based reservations are not supported when exporting a RBD image
-        through more than one iSCSI gateway.
+        The Ceph iSCSI gateway does not support reservations in any scenario.
+        ``tcmu-runner`` does not support reservations.
+
+    .. warning::
+
+        The iSCSI code is in maintenance as of November 2022. This means that
+        it is no longer in development, it is feature complete, and it will not
+        be updated to add new features. In this context, the fact that the
+        iSCSI code is in maintenance means that it does not support
+        reservations and it will not support reservations.
```

Comment:
The last sentence about reservations should be removed and I'd move the remainder of this node is to the iSCSI overview page (`doc/rbd/iscsi-overview.rst`).

Reply:
I've moved this to `doc/rbd/iscsi-overview-rst`. You're correct. That's a better place for it.